src/dr_trigger.py

```diff
@@ -141,3 +141,19 @@ def trigger_dump(s_pid, s_port, path, s_flow_ptr):
     dump_file.close()
     sock.close()
     return path
+
+def main():
```

Comment:
is this change needed for dr_ste_v0?

Reply:
no in dr_ste_v0 a different mechanism was used there